mne/evoked.py

```diff
@@ -1091,7 +1091,10 @@ def _read_evoked(fname, condition=None, kind='average', allow_maxshield=False):
         else:
             # Put the old style epochs together
             data = np.concatenate([e.data[None, :] for e in epoch], axis=0)
-        data = data.astype(np.float64)
+        if np.isrealobj(data):
+            data = data.astype(np.float64)
+        else:
+            data = data.astype(np.complex128)
```

Comment:
These lines have no corresponding test. Somewhere in `mne/tests/test_evoked.py` you could do something simple like:
```
evoked_complex = evoked.copy()
evoked_complex.data = 1j * evoked_complex.data
fname_temp = str(tmpdir.join('complex-ave.fif'))
evoked_complex.save(fname_temp)
evoked_complex = read_evokeds(fname_temp)[0]
assert_allclose(evoked.data, evoked_complex.imag)
```
or just overwrite `evoked.data` with random complex data, etc. Up to you

Reply:
thx, has been added now!